### DISCUSSION TITLE
Restrict selecting 1-10 Keys mods for non-converted beatmaps (mania specific)

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#22840] [Q&A]
The user shouldn't see this mods at all or see them dimed and non-selectable.

![image](https://user-images.githubusercontent.com/86934170/224510298-3394629e-43c8-45f1-9610-5483969a3305.png)

**Comment by ItsShamed** [comment]
FYI the “dimmed” mods are not “non-selectable” at all, they will just unselect the incompatible mods upon clicking on them. Also, they _need_ to be separate mods as it is how they are identified by https://github.com/ppy/osu-web, using a slider won't work really well.

**Reply by Loreos7** [reply]
Screenshot is just for an example of how this would probably look like
I know they're incompatible

**Reply by ItsShamed** [reply]
Uhh then I'm not quite sure that I've understood your issue correctly

**Reply by Loreos7** [reply]
Applying 1-10 Keys mods to mania specific beatmaps doesn't change anything. I can't play beatmap with 4k if its difficulties are for 8k only. But these mods, especially for newbies, a bit confusing and feel like not-working if a player have an ability to apply them.
So my suggestion is to hide them for mania specific beatmaps or make them grey and unselectable (because these mods are for converted beatmaps only)

**Comment by peppy** [comment]
Either that, or they should be made to work with converting from other key modes (I've always wondered why this isn't a thing to be honest). Arguable whether it's required but one or the other. It's very confusing as it stands.

**Reply by Loreos7** [reply]
If every ranked map would be playable 1-10K alike that would be insane. Should ask mania players first though.
